osu.Game/Graphics/Containers/OsuRearrangeableListContainer.cs

```diff
@@ -18,11 +23,44 @@ public abstract class OsuRearrangeableListContainer<TModel> : RearrangeableListC
 
         protected override ScrollContainer<Drawable> CreateScrollContainer() => new OsuScrollContainer();
 
+        private Sample sampleSwap;
+        private double sampleLastPlaybackTime;
+
         protected sealed override RearrangeableListItem<TModel> CreateDrawable(TModel item) => CreateOsuDrawable(item).With(d =>
         {
             d.DragActive.BindTo(DragActive);
         });
 
         protected abstract OsuRearrangeableListItem<TModel> CreateOsuDrawable(TModel item);
+
+        protected OsuRearrangeableListContainer()
+        {
+            Items.CollectionChanged += (_, args) =>
+            {
+                if (args.Action == NotifyCollectionChangedAction.Move)
+                    playSwapSample();
+            };
```

Comment:
Is there a particular reason why this is done in ctor? Usual practice is to put these in `LoadComplete()`.